tests/download_test.py

```diff
@@ -28,6 +29,18 @@ def test_download(self):
                     expected_return_code=0,
                 )
 
+    def test_download_no_results(self):
+        with file_utils.temp_dir(dict(malicious=self.test_eicar)) as (path, files):
```

Comment:
Same question about eicar, what's the use case here?

Reply:
Additionally, we should be making sure to `base64` encode eicar at a minimum, as we don't want `polyswarm-cli` to potentially trigger antivirus when people are checking out the repo or installing it.

Reply:
Ideally I think we'd use something other than eicar here for these tests. I don't think it's necessary and there's still a risk that AVs would detect it while encoded.

Reply:
All the mocks were generated using `e2e`, and to get the results from a detection I used `eicar`. I guess it would be ok to remove it and we could generate artificial detection cases manually, but this would greatly increase the burden of fixing tests if we change stuff in the future, as they would stop working in `e2e`. We could possibly change `e2e` to detect some other stuff rather than `eicar`, something that anti-viruses would no complain, and write a special microengine that detects this other stuff. That is probably a good compromise.

Reply:
These are all mockups (either for what the mocked `polyswarm-api` method returns (unit tests) or for what the mocked API resource returns (integration tests). It doesn't really matter for these test scenarios what it is returned (could be anything, as long as it is `bytes`). We could put any content. The test cases only check that the content is processed and presented correctly by `polyswarm-cli`.